src/global.ts

```diff
@@ -263,17 +263,26 @@ export interface LogoutOptions {
   /**
    * The URL where Auth0 will redirect your browser to after the logout.
    *
-   * > Note that if the `client_id` parameter is included, the
+   * **Note**: If the `client_id` parameter is included, the
    * `returnTo` URL that is provided must be listed in the
    * Application's "Allowed Logout URLs" in the Auth0 dashboard.
    * However, if the `client_id` parameter is not included, the
    * `returnTo` URL must be listed in the "Allowed Logout URLs" at
    * the account level in the Auth0 dashboard.
+   *
+   * If the `returnTo` URL is **not** set and `client_id` is **not** `null`, the server returns the user to the first Allowed Logout URLs set in the Dashboard.
```

Comment:
Documenting the behavior of sending/not sending `client_id` is OK. But how this line is worded, `returnTo` seems like a required parameter and I see below that it's not. The more we explain a property the more we risk not keeping these docs up to date with the official docs. Also, I could think, why would I care reading these docs if I'm not setting this property at all? I think we should word it assuming this `returnTo` value is present by now. This may be translated to, this line "both not null - scenario" line not necessary.

Reply:
Good shout - I've removed that line and just linked off to further info.